rust/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "codec-fixtures"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+libipld = "0.13.0"
```

Comment:
Can we float this? note that in the Actions runner for Node.js we `npm install $(node -pe 'Object.keys(require("./package.json").dependencies).map((d) => `${d}@latest`).join(" ")');` and in Go we `go get github.com/ipld/go-ipld-prime@master; go get -u -d` so we're always testing on whatever is sitting at the tip and the Cron job will alert us when something broken in one of those stacks.

Reply:
Oh, this is what the cron job was about, I didn't get that. I've updated the workflow, I hope it now works the same way as for the other libs.